src/ui/components/UploadingScreen.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from "classnames";
+import React, { useEffect, useState } from "react";
+import Modal from "ui/components/shared/NewModal";
+
+const slides = [
+  {
+    header: "Success!",
+    content: (
+      <>
+        <div>{`Your Replay has been recorded! 🎉`}</div>
+        <div>{`We're uploading and will let you know when it's safe to close this tab.`}</div>
+      </>
+    ),
+  },
+  {
+    header: "Our apologies",
+    content: (
+      <>
+        <div>{`Your upload appears to have stalled.`}</div>
+        <div>{`We're sorry for the inconvenience and we're looking into the issue. In the meantime, please try your recording again.`}</div>
+      </>
+    ),
+  },
+];
+
+export default function UploadingScreen() {
+  const [current, setCurrent] = useState<number>(1);
+  const { header, content } = slides[current - 1];
+
+  useEffect(() => {
+    // Show the error content at the 1 minute mark.
+    setTimeout(() => {
+      setCurrent(2);
+    }, 60000);
+  }, []);
```

Comment:
The lifecycle's irrelevant for now since we yank the user from this screen (`replay.io/view?loading=true`) and navigate them to the new replay.

Different story once we move to a single URL loading flow, but in that case, this would become obsolete anyway since we'd pop open the Upload Form in its place immediately.